source/MaterialXView/Main.cpp

```diff
@@ -197,9 +197,14 @@ int main(int argc, char* const argv[])
     mx::FilePath installSearchPath = mx::FilePath::getModulePath().getParentPath();
     mx::FilePath devSearchPath = installSearchPath.getParentPath().getParentPath().getParentPath();
     searchPath.append(installSearchPath);
-    if (!devSearchPath.isEmpty() && (devSearchPath / "libraries").exists())
+    if (!devSearchPath.isEmpty() && devSearchPath.exists())
     {
         searchPath.append(devSearchPath);
+        devSearchPath = devSearchPath / "libraries";
+        if (devSearchPath.exists())
+        {
+            searchPath.append(devSearchPath);
+        }
```

Comment:
We need to add two paths here now since the hard coded "libraries" was removed from source code searches inside shadergen.
1. <installpath> - for getting resources
2. .<installpath>/libraries - for getting shader source code